src/types/singleModel.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+
+'use strict';
+
+const BaseType = require('./base');
+
+class SingleModel extends BaseType {
+
+    constructor (parentModel: Object, Model: Function, options: Object = {}) {
+        super(parentModel, options);
+
+        Object.assign(this, {
+            _ModelConstructor: Model,
+            _model: new Model(parentModel),
```

Comment:
This could link a model to a model that doesn't own it. When an Event gets instantiated,this would instantiate a new Place scoped to the Event, not the Exposition, and not linked to the Performer with the same ID in another event.